bears/rest/reSTLintBear.py

```diff
@@ -14,13 +28,17 @@ class reSTLintBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
 
-    def run(self, filename, file):
+    def run(self, filename, file, ignore_unknown_roles: bool=False):
```

Comment:
I dont see the benefit of replicating `rstcheck` , unless we know of a feature in `restlint` that is missing from `rstcheck`...?